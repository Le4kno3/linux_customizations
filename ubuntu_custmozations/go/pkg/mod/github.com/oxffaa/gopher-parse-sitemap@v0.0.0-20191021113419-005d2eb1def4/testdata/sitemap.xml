<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
  <url>
    <loc>http://HOST/</loc>
  </url>
  <url>
    <loc>http://HOST/tools/</loc>
    <lastmod>2015-05-07T19:13:09+09:00</lastmod>
  </url>
  <url>
    <loc>http://HOST/contribution-to-oss/</loc>
    <lastmod>2015-05-07</lastmod>
    <changefreq>monthly</changefreq>
  </url>
  <url>
    <loc>http://HOST/page-1/</loc>
    <lastmod>2015-05-07T19:13:09+09:00</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.9</priority>
  </url>
</urlset>