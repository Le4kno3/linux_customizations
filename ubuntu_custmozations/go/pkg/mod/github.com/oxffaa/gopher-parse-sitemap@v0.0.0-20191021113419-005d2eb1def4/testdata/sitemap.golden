http://HOST/
always
0.5
http://HOST/tools/
2015-05-07T19:13:09+09:00
always
0.5
http://HOST/contribution-to-oss/
2015-05-07T00:00:00Z
monthly
0.5
http://HOST/page-1/
2015-05-07T19:13:09+09:00
monthly
0.9
